Sources/NSManagedObjectContext+Persistence.swift

```diff
@@ -17,72 +17,73 @@ public extension NSManagedObjectContext {
         delete(object)
     }
     
-    func deleteAll<T: NSManagedObject>(_ entity: T.Type, predicate: NSPredicate? = nil) {
-        let objects = fetchAll(entity, predicate: predicate)
+    func deleteAll<T: NSManagedObject>(_ entity: T.Type, filteredBy predicate: NSPredicate? = nil) {
+        let objects = fetchAll(entity, filteredBy: predicate)
         
         for object in objects {
```

Comment:
We could even use a higher order function and write this like:

```swift
objects.forEach { delete($0) }
```

P.S. I don't have a strong opinion for this. Slightly verbose way is sometimes clearer.